code/game/objects/items/weapons/stunbaton.dm

```diff
@@ -243,9 +243,9 @@
 	force = 3
 	throwforce = 5
 	knockdown_duration = 6 SECONDS
-	stam_damage = 40 // three hits to crit
```

Comment:
Perhaps you should have tested it after all,

Reply:
no this is now how it works

Reply:
OH WAIT ITS NOT THE PARENT OF BATON LMAO IM DUMB

Reply:
removing the stam damage, means that it inherits the stamina damage from the stun baton, EG 60